client/src/pages/Cart.tsx

```diff
@@ -1,14 +1,22 @@
+import { useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { QueryClient, QueryClientProvider } from 'react-query';
 import CartNavBar from '../components/Cart/CartNavBar';
 import CartList from '../components/Cart/CartList';
 import DownComponent from '../components/Cart/DownComponet';
 
+const queryClient = new QueryClient();
```

Comment:
Why did you add a new query client class? Farah initialized one in the main.tsx and added its provider to all the components.